Week-2/Assignment-2/auto-loan.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="auto-loan.css" />
+    <title>Loan Offers</title>
+  </head>
+  <body>
+    <h1>Comparison of Dealer Incentives and Loan Offers</h1>
+
+    <div class="container">
```

Comment:
Here,
1. Please wrap all your content in a parent tag.
2. The `Heading` and the `content` is not sync with each other.